org.unitystation.StationHub.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=StationHub
+Comment=Connect to Unitystation servers & manage your installs.
```

Comment:
Just a small nitpick: Comments in desktop files are usually without dot at the end of the sentence.